### PR TITLE
chris' comments

### PR DESCRIPTION
**Pull request by cfl0ws**

### CODE REVIEW

File: gitbook/participation/dry-run-5.md [hunk 1]
```diff
@@ -1,33 +1,36 @@
...
 
 ## Cluster Parameters:
 
-* GPUs optional
-* Solana software version: v0.20.0
+* GPUs optional [CJR: Do validators running GPUs have a better chance of surviving?]
+* Solana software version: v0.20.0 [CJR: Should be updated in the FAQ]
 * Target transaction rate of first round: 2,000 TPS
 * Transaction rate round increment: 2,000 TPS
+* Round duration: 20 minuts
```
Comment (CriesofCarrots) [inline gitbook/participation/dry-run-5.md:13]: minutes

File: gitbook/participation/dry-run-5.md [hunk 2]
```diff
@@ -1,33 +1,36 @@
...
+Dry Run 5 will test the performance of a heterogeneous cluster of validators under heavier transaction load. It will be structured as a series of increasing rounds of transactions per second \(TPS\). At the end of each round the validators that survive receive additional stake for the next round. [CJR: What would cause a validator to not survive?] The rounds will continue until the cluster stops making progress.
 
 ![Ramp TPS rounds visualized](../.gitbook/assets/image%20%282%29.png)
 
 ## Cluster Parameters:
 
-* GPUs optional
-* Solana software version: v0.20.0
+* GPUs optional [CJR: Do validators running GPUs have a better chance of surviving?]
+* Solana software version: v0.20.0 [CJR: Should be updated in the FAQ]
```
Comment (mvines) [inline gitbook/participation/dry-run-5.md:10]: What FAQ do you mean?  It's updated on doc.solana.com

File: gitbook/participation/dry-run-5.md (hunk 1 above)
Comment (mvines) [reply gitbook/participation/dry-run-5.md:13]: I just added this to the doc

File: gitbook/participation/dry-run-5.md [hunk 3]
```diff
@@ -1,33 +1,36 @@
...
 
-The new validator stake will take 3-4 epochs \(~2 hours\) to warmup. During this time the stake of the Solana boot nodes will be reduced as well.
+The new validator stake, delegated in Phase 1, will take 3-4 epochs \(~2 hours\) to warmup. During this time the stake of the Solana boot nodes will be reduced as well.
 
-### Epoch 14+: Ramp TPS rounds begin! \(approx. 1-2 hours per round\)
+### Phase 3: Epoch 14+: Ramp TPS rounds begin! \(approx. 1-2 hours per round\)
 
 Now that all validator stake is active, the Ramp TPS program will begin running the rounds of increasing transactions-per-second until the cluster dies. Each round starts with of 20 minutes of solid transactions. After 20 minutes all validators that remain with the cluster will receive an additional stake delegation. This new stake will also take 3-4 epochs to warm up, and once warm up is complete the next round commences with an increased transaction rate.
 
+[CJR: Should validators try and re-stake the new delegation? Can that stake be delegated right away or does it have to wait for the warm up period to complete?]
```
Comment (mvines) [inline gitbook/participation/dry-run-5.md:32]: The new stake is automatically added to all surviving validators by the ramp tool, the validators themselves don't have access to the stake.  But yes the new stake is delegated immediately

File: gitbook/participation/dry-run-5.md [hunk 4]
```diff
@@ -1,33 +1,36 @@
 # Dry Run 5
 
-Dry Run 5 will test the performance of a heterogeneous cluster of validators under heavier transaction load. It will be structured as a series of increasing rounds of transactions per second \(TPS\). At the end of each round the validators that survive receive additional stake for the next round. The rounds will continue until the cluster stops making progress.
+Dry Run 5 will test the performance of a heterogeneous cluster of validators under heavier transaction load. It will be structured as a series of increasing rounds of transactions per second \(TPS\). At the end of each round the validators that survive receive additional stake for the next round. [CJR: What would cause a validator to not survive?] The rounds will continue until the cluster stops making progress.
```
Comment (mvines) [inline gitbook/participation/dry-run-5.md:3]: Some combo of hardware, network, software.   Partially under each validator's control, but also hitting some software bug is also a possibility

**Pull request closed without merging**